README.md

```diff
@@ -1,92 +1,35 @@
-2. Log in as an Administrator on Confluence to configure the plugin.
-3. Create a new app in your Confluence Server by going to **Settings > Apps > Manage Apps**. For older versions of Confluence, go to **Administration > Applications > Add-ons > Manage add-ons**.
-4. Choose **Settings** at the bottom of the page, enable development mode, and apply the changes. Development mode allows you to install apps from outside of the Atlassian Marketplace.
-5. Press **Upload app**.
-
-    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/158bd7b7-4e36-41d5-872a-def2f617213f)
-
-6. Upload the OBR file you downloaded earlier. 
-7. Once the app is installed, press **Configure** to open the configuration page.
-8. In the **Webhook URL field**, enter the URL that is displayed after you typed `/confluence install server` - it is unique to your server. You'll need to copy/paste it from Mattermost into the **From this URL** field in Confluence. 
-9. Press **Save** to finish the setup.
-10. Go to **Settings > Apps > Manage Apps**.
-11. Choose **Settings** at the bottom of the page, enable development mode, and apply the change. Development mode allows you to install apps from outside of the Atlassian Marketplace.
-
-    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/fa27854e-6305-4164-963f-a5692284bf95)
-
-12. Once installed, you will see the "Installed and ready to go!" message in Confluence.
-13. You can now go to Mattermost and type `/confluence subscribe` in the channel you want to get notified by Confluence.
-
-#### Set up Confluence Cloud
-
-To get started, type in `/confluence install cloud` in a Mattermost chat window.
-
-1. Log in as an Administrator on Confluence.
-2. Go to **Settings > Apps > Manage Apps**.
-3. Choose **Settings** at the bottom of the page, enable development mode, and apply the change. Development mode allows you to install apps from outside of the Atlassian Marketplace.
-
-    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/34471b0f-e54d-476c-b843-bf71355b6831)
-
-4. Press **Upload App** button.
-
-    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/2ca5cc02-8a98-462d-b5f8-1cb20b7272d6)
-
-5. In **From this URL**, enter the URL that is displayed after you typed `/confluence install cloud` - it is unique to your server. You'll need to copy/paste it from Mattermost into the **From this URL** field in Confluence. 
-6. Once installed, you will see the "Installed and ready to go!" message in Confluence.
-7. You can now go to Mattermost and type `/confluence subscribe` in the channel you want to get notified by Confluence.
-
 ## Configure notifications
 
 ### /confluence subscribe
```

Comment:
Not sure if this is an admin-only feature, but either way it seems like it's the only feature so it makes sense to be separate from the admin guide :+1:

Reply:
done